src/chrome/content/rules/ORF.at.xml

```diff
@@ -118,33 +131,25 @@
 	<target host="appfeeds.orf.at" />
 	<target host="audioapi.orf.at" />
 	<target host="autodiscover.orf.at" />
-	<target host="bernard.orf.at" />
 	<target host="bert.orf.at" />
-	<target host="bianca.orf.at" />
 	<target host="bikini.orf.at" />
 	<target host="bits.orf.at" />
 	<target host="boa.orf.at" />
 	<target host="bolek.orf.at" />
 	<target host="book.orf.at" />
 	<target host="booking.orf.at" />
 	<target host="bpo.orf.at" />
-	<target host="brain.orf.at" />
 	<target host="buergerforumdb.orf.at" />
 	<target host="cloud.orf.at" />
-	<target host="clyde.orf.at" />
-	<target host="content.orf.at" />
```

Comment:
Please add a comment for it

Reply:
content.orf.at redirects to https, then to a microsoft-cloud-login (which I presume only orf employees have access). what kind of comment should I add for that? the commit message mentions already that it redirects itself.

Reply:
You can add it to the `Cert expired` section